src/StreamVersion.java

```diff
@@ -0,0 +1,25 @@
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class StreamVersion {
+    public static void fizzBuzz() {
+        IntStream.rangeClosed(1, 100)
+                .mapToObj(x -> x % 3 == 0 ? (x % 5 == 0 ? x + " FizzBuzz" : x + " Fizz") : (x % 5 == 0 ? x + " Buzz" : x))
```

Comment:
That is an interesting solution and I would like to go over this exercise on one of our lessons.

Reply:
This is also stream, so i dont know why you created second method.

Reply:
I created the second method because I didn't know how to not display the remaining numbers in this method.